src/main/java/teammates/storage/entity/AdminEmail.java

```diff
@@ -98,7 +88,7 @@ public void setSendDate(Date sendDate) {
     }
 
     public String getEmailId() {
-        return this.emailId;
+        return Key.create(AdminEmail.class, this.emailId).toWebSafeString();
```

Comment:
Is there a benefit in letting the retrieved Id be the string of the key instead of the actual Id? I am a little concerned that it may be confusing to developers in the future as to why the `emailId` of AdminEmail and AdminEmailAttributes differs and why only certain Entity classes are treated as such.